lib/cretonne/src/ir/instructions.rs

```diff
@@ -120,6 +120,7 @@ pub enum InstructionData {
     },
     Ternary { opcode: Opcode, args: [Value; 3] },
     MultiAry { opcode: Opcode, args: ValueList },
+    NullAry { opcode: Opcode },
```

Comment:
Ha, it's no that long ago I deleted this format because there were no uses left, but it's the right thing for a pop.